docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,929 @@ The LDML specification is divided into the following parts:
+
+```xml
+<namePattern>{given-initial}.({given2}) {surname}</namePattern>
+<namePattern alt=”2”>{given-initial}. {surname}</namePattern>
+```
+
+The first pattern would be used when the “given2” field is populated, and the second pattern would be used when the “given2” field is empty.
+
+Rules 1 and 3 can conflict in similar ways. If the personName element contains (there’s a space between the period and the opening parenthesis again):
+
+```xml
+<namePattern>{given-initial}. ({given2}) {surname}</namePattern>
+```
+
+And the input name record contains:
+
+| `given` | `given2` | `surname` |
+| ------- | -------- | --------- |
+|         | Bar      | Baz       |
+
+The output is:
+
+    Bar) Baz
+
+Because the “given” field is empty, rule 1 not only has us delete it, but also all punctuation up to “{given2}”. This includes _both_ the period _and_ the opening parenthesis. Again, to solve this, you’d supply two namePatterns:
+
+```xml
+<namePattern>{given-initial}. ({given2}) {surname}</namePattern>
+<namePattern alt=”2”> ({given2}) {surname}</namePattern>
+```
+
+The output would then be:
+
+    (Bar) Baz
+
+The first namePattern would be used if the “given” field was populated, and the second would be used if it was empty.
+
+If, instead, the input name record contains:
+
+| `given` | `given2` | `surname` |
+| ------- | -------- | --------- |
+| Foo     |          | Baz       |
+
+The output is:
+
+    F. Baz
+
+### 6.7 <a name="6-7-deriving-initials" href="#6-7-deriving-initials">Deriving initials</a>
+
+The following process is used to produce initials when they are not supplied by the name object.
+
+| Action              | Result |
+| ------------------- | ------ |
+| 1. Split into words | “Mary” and “Beth” |
+| 2. Each converted to initials | “M” and “B” |
+| 3. The ***initial*** pattern is applied to each<br>`  <initialPattern type="initial">{0}.</initialPattern>` | “M.” and “B.” |
+| 4. Finally recombined with ***initialSequence***<br>`  <initialPattern type="initialSequence">{0} {1}</initialPattern>` | “M. B.” |
+
+See the “initial” modifier in the [Modifiers](#5-2-modifiers) section for more details.
+
+### 6.8 <a name="6-8-handling-foreign-names" href="#6-8-handling-foreign-names">Handling foreign names</a>
+
+There are two main challenges in dealing with foreign name formatting that needs to be considered. One is the ordering, which is dealt with under the section [[2.3 nameOrderLocales Element](#2-3-nameorderlocales-element)]. The other is spacing.
+
+Some writing systems require spaces (or some other non-letters) to separate words. For example, [Hayao Miyazaki](https://en.wikipedia.org/wiki/Hayao_Miyazaki) is written in English with given name first and with a space between the two name fields, while in Japanese there is no space with surname first: 
+
+    [宮崎駿](https://ja.wikipedia.org/wiki/%E5%AE%AE%E5%B4%8E%E9%A7%BF)
```

Comment:
Not sure why this isn't interpreted right in markdown

Reply:
Changed to use block quote instead of indent (which implies code block)